bool command    # True for opening gripper, False for closing it
float32 percentage   # gripper fingers percentage
